src/bin/pg_autoctl/cli_enable_disable.c

```diff
@@ -425,8 +559,10 @@ cli_disable_maintenance(int argc, char **argv)
 
 	if (!monitor_wait_until_node_reported_state(
 			&(keeper.monitor),
+			keeper.config.formation,
+			keeper.config.groupId,
 			keeper.state.current_node_id,
-			CATCHINGUP_STATE))
+			SECONDARY_STATE))
```

Comment:
The reason we change that here is because we now display the transitions as they are happening. In the master branch we did not, so we exited as soon as possible. In this PR we think it's more useful to show an extra transition even if that means waiting longer.